### ISSUE TITLE
CI: update Github actions

### ISSUE THREAD

**Issue by MSoegtropIMC** [coq/platform#359]
We get the CI warning:

CI: Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/checkout@v2, actions/upload-artifact@v2. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. (delayed, creates issue)

**Comment by Zimmi48** [comment]
We should try bumping to actions/checkout@v3 as this may be enough to fix this warning.

**Comment by MSoegtropIMC** [comment]
Yes, this is something planned after the current release is through.
